src/Microsoft.Health.Dicom.SqlServer/Features/Indexing/SqlReindexStore.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.Dicom.Core.Features.Indexing;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.Indexing
+{
+    internal class SqlReindexStore : IReindexStore
```

Comment:
Do we need this class if nothing is implemented?

Reply:
Also, these don't look specific to Reindex, can these be more generic like GetJobEntriesAsync(), UpdateJobEntriesAsync()...

Reply:
The fake implementation is for other people to be able to run the application without injection failure, so that they won't be blocked.

This table stores `Reindex` status instead of Job information. It has `TagKey`, `StartWatermark`,`EndWatermark`,`OperationId`, `Failures`, (we may add more), which describe current Reindex status on this tag.

BTW, we use `Operation` to replace `Job`, since feel it make more sense for customer. 

We are discussing merging `Reindex` status and `Deindex` status into one single table (maybe call it `SqlIndexStore` or `SqlIndexStatusStore`?), but I have some concerns since `Reindex` and `Deindex` have different set of parameters -- e.g  `Reindex` need `StartWatermark` and `EndWatermark` which are useless for `Deindex`.

Reply:
Check with Will again, we discussed to simplify this. You should maybe add a todoapi and wait for the service implementor to add the right methods

Reply:
removed.